src/bazel/pkg_config_repository.bzl

```diff
@@ -86,6 +86,13 @@ def _exec_pkg_config(repo_ctx, flag):
         print("pkg-config is not found")  # buildifier: disable=print
         return []
     result = repo_ctx.execute([binary, flag] + repo_ctx.attr.packages)
+
+    # If result.stdout.strip() is empty, pkg-config will be re-executed with
+    # the --keep-system-cflags option added. Typically, -I/usr/include is
+    # returned, enabling bazel to recognize packages as valid even when
+    # pkg-config does not output cflags with standard options.
+    if result.stdout.strip() == "":
+        result = repo_ctx.execute([binary, flag] + ["--keep-system-cflags"] + repo_ctx.attr.packages)
```

Comment:
Should `--keep-system-cflags` be applied only when the output of `--cflags-only-I` is empty?  If yes, please move this logic after `_exec_pkg_config(repo_ctx, "--cflags-only-I")` in `_pkg_config_repository_impl`.

It's fine to modify `flag` to `flags` that takes a list of string.

Reply:
How about a modification like this?

Reply:
It seems that the CI check has also passed.

Reply:
Thank you for the update.
I'm running the internal process now. Please keep this PR as-is.